addons/website/views/snippets.xml

```diff
@@ -365,6 +365,74 @@
     </section>
 </template>
 
+<template id="s_company_team" name="Company Team">
+    <section class="s_company_team">
+        <div class="container">
+            <div class="row mb32">
+                <div class="col-md-12 col-xs-12">
+                    <h1 class="text-center">
+                        Meet the Executive Team
+                    </h1>
+                </div>
+            </div>
```

Comment:
Pretty sure the `col-xs-12` (and all of them below) is useless. Also are these resizable titles back? :) I had to remove them a while ago... :)